config.md

```diff
@@ -388,7 +388,7 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 This information MAY be structured or unstructured.
 Annotations MUST be a key-value map where both the key and value MUST be strings.
 While the value MUST be present, it MAY be an empty string.
-Keys MUST be unique within this map, and best practice is to namespace the keys.
+Keys MUST be unique and MUST NOT be an empty string within this map, and best practice is to namespace the keys.
```

Comment:
I'd rather not split “unique within this map”.  How about splitting each idea into its own sentence:

> Keys MUST be unique within this map.
> Keys MUST NOT be an empty string.

We can drop the “best practice” portion, because the following line is “Keys SHOULD be named using a reverse domain notation…”, which covers that idea more explicitly.  If we want to give explicit namespacing advice to annotation-spec devs without access to a domain name, I'd rather address that directly (in a separate PR?).